_includes/pbc/all/install-features/202410.0/install-the-product-labels-feature.md

```diff
@@ -0,0 +1,637 @@
+
```

Comment:
This page was carried over from previous versions, introduced changes for `ProductLabelLocalizedAttributesWritePublisherPlugin` .